src/map.cpp

```diff
@@ -3341,7 +3341,7 @@ ter_id map::get_roof( const tripoint &p, const bool allow_air )
 
     const auto &ter_there = ter_at( p );
     const auto &roof = ter_there.roof;
-    if( roof.empty() || roof == null_ter_t ) {
+    if( roof.is_null() ) {
```

Comment:
I know about the operator, but I didn't know whether it's more desirable, than explicit `is_null()`. The current code uses `is_null()` almost everywhere so I decided to tick to it.  I'm glad to know, that shorter syntax is more preferable here and will replace all relevant occurrences.